src/main/java/co/alphavantage/input/exchange/Currency.java

```diff
@@ -0,0 +1,62 @@
+package co.alphavantage.input.exchange;
+
+import co.alphavantage.input.ApiParameter;
+
+/**
+* Interval parameter, i.e interval=15min
+*/
+public class Currency {

```

Comment:
This looks a bit complicated to me. Why do we need the Form, To and the FromTo classes?